### PR TITLE
Add external Mime db custom json mime-types.

### PR DESCRIPTION
**Pull request by Coderrob**
- Add additional unit tests to verify logic in TT
- Add MimeDb external source to the generated known mime-types
- Update Text project SDK format

Resolves issue https://github.com/zone117x/MimeMapping/issues/8

### CODE REVIEW

**Comment by Coderrob** [comment]
@zone117x looks like I can't target .Net Standard 2.1 for .Net Core 3.0 inclusion. Is that alright to bring in?

File: KnownMimeTypes.cs [hunk 1]
```diff
@@ -5149,1825 +4326,80 @@ public static class KnownMimeTypes
...
...case _ext_silo:					return Msh;				case _ext_dae:					return Dae;				case _ext_dwf:					return Dwf;				case _ext_gdl:					return Gdl;				case _ext_gtw:					return Gtw;				case _ext_mts:					return Mts;				case _ext_vtu:					return Vtu;				case _ext_wrl:				case _ext_vrml:					return Wrl;				case _ext_x3db:				case _ext_x3dbz:					return X3db;				case _ext_x3dv:				case _ext_x3dvz:					return X3dv;				case _ext_x3d:				case _ext_x3dz:					return X3d;				case _ext_appcache:				case _ext_manifest:					return Appcache;				case _ext_ics:				case _ext_ifb:					return Ics;				case _ext_css:					return Css;				case _ext_csv:					return Csv;				case _ext_html:				case _ext_htm:				case _ext_shtml:					return Html;				case _ext_n3:					return N3;				case _ext_txt:				case _ext_text:				case _ext_conf:				case _ext_def:				case _ext_list:				case _ext_log:				case _ext_in:				case _ext_ini:					return Txt;				case _ext_dsc:					return Dsc;				case _ext_rtx:					return Rtx;				case _ext_sgml:				case _ext_sgm:					return Sgml;				case _ext_tsv:					return Tsv;				case _ext_t:				case _ext_tr:				case _ext_roff:				case _ext_man:				case _ext_me:				case _ext_ms:					return T;				case _ext_ttl:
```
Comment (zone117x) [inline KnownMimeTypes.cs]: Looks like some TT whitespace changes causes this switch case statement to be one giant line. Can you fix this?

File: KnownMimeTypes.tt [hunk 2]
```diff
@@ -34,46 +37,57 @@ namespace MimeMapping
...
+			var resource = JObject.Parse(content);
+            return from mimeTypes in resource.Children() 
+                let mimeType = ((JProperty) mimeTypes).Name 
+                from mimeTypeProperties in mimeTypes.Children() 
+                from mimeTypeProperty in mimeTypeProperties.Children() 
+                where (mimeTypeProperty as JProperty)?.Name == "extensions" 
+                select new[]
+                {
+                    mimeType, 
+                    mimeTypeProperty.Values().First().Value<string>()
```
Comment (zone117x) [inline KnownMimeTypes.tt]: Is it safe to only use the first entry here? I glanced over the json and the only notable missing extension I saw was in this entry:
```
  "text/yaml": {
    "extensions": ["yaml","yml"]
  },
```

Comment (Coderrob) [reply KnownMimeTypes.tt]: Good catch! Will address.

File: KnownMimeTypes.cs (hunk 1 above)
Comment (Coderrob) [reply KnownMimeTypes.cs]: Saw that too... will do!

**Comment by Coderrob** [comment]
@zone117x updated with requested changes. Formatting looks good now. Made another tweak to make pulling Json files a little more generic to support other external sources.

**Pull request merged**